src/locales/es/battle.ts

```diff
@@ -131,5 +131,6 @@ export const battle: SimpleTranslationEntries = {
   "battlerTagsSaltCuredOnAdd": "{{pokemonNameWithAffix}} is being salt cured!",
   "battlerTagsSaltCuredLapse": "{{pokemonNameWithAffix}} is hurt by {{moveName}}!",
   "battlerTagsCursedOnAdd": "{{pokemonNameWithAffix}} cut its own HP and put a curse on the {{pokemonName}}!",
```

Comment:
```suggestion
  "battlerTagsSaltCuredOnAdd": "¡{{pokemonNameWithAffix}} está en salazón!",
  "battlerTagsSaltCuredLapse": "¡{{moveName}} ha herido a {{pokemonNameWithAffix}}!",
  "battlerTagsCursedOnAdd": "¡{{pokemonNameWithAffix}} reduce sus PS y maldice a {{pokemonName}}!",
```

![image](https://github.com/pagefaultgames/pokerogue/assets/170796740/319f81cd-51a7-4365-874f-15de26e5cf0d)